app/controllers/refine/inline/criteria_controller.rb

```diff
@@ -88,8 +87,21 @@ def destroy
     handle_filter_update(@refine_filter.to_stable_id)
   end
 
+  def clear
+    puts "CLEARING filter"
```

Comment:
these `puts` lines meant for live action, or dev work?

Reply:
Ah good catch. Yes this is leftover. Will clean that up quick sorry